mono/mini/mini.h

```diff
@@ -2069,6 +2074,12 @@ guint     mini_type_to_stind                (MonoCompile* cfg, MonoType *type);
 MonoJitInfo* mini_lookup_method             (MonoDomain *domain, MonoMethod *method, MonoMethod *shared);
 guint32   mono_reverse_branch_op            (guint32 opcode);
 void      mono_disassemble_code             (MonoCompile *cfg, guint8 *code, int size, char *id);
+// Subset of MonoJumpInfo.
+ typedef struct MonoJumpInfoTarget {
+	MonoJumpInfoType type;
+	gconstpointer   target;
+} MonoJumpInfoTarget;
+MonoJumpInfoTarget mono_call_to_patch       (MonoCallInst *call);
```

Comment:
This should be next to MonoJumpInfo not in the middle of function declarations.

Reply:
I tried. It didn't work. I'll try again.
Originally I had not struct, just two out params, which would work.

Reply:
I mean just move it next to the declaration of MonoJumpInfo, don't define it in the middle of functions.

Reply:
Next to MonoJumpInfo in `mini-runtime.h`, seems best, tried, failed, will look again.

Among types in `mini.h`, away from functions, seems marginally better, might satisfy, will do if first fails and satisfies.

Reply:
Several choices here:

1. Put it in `mini-runtime.h` near `MonoJumpInfo`. Change `mini.h` to `#include mini-runtime.h`.
2. Put in `mini.h` among types instead of functions.
3. Use an "out return param pointer" instead of returning struct by value, then can move it to `mini-runtime.h` w/o an extra `include`.

I don't like adding `#include` so didn't do 1.
But of course I'm open to any of them.